.github/workflows/docker-images-nightly.yml

```diff
@@ -76,7 +98,7 @@ jobs:
           push: true
```

Comment:
hopefully the build cache will kick in, but just to be in the safer side, should we perhaps do a separate build step, and then 2 separate push steps?

Reply:
Fixed as recommended [here](https://docs.docker.com/build/ci/github-actions/copy-image-registries/).